src/app/api/catalogs/models.ts

```
@@ -115,7 +111,47 @@ function createPlaylistId(channel: string) {
   return channel.substring(0, 1) + "U" + channel.substring(2);
 }
 
-async function getPlaylistItems(channel: any) {
+// TODO: Clean this file up, it is an eyesore 😵
+async function getPlaylistVideos(playlist: any) {
+  let playlistItemData: VideoMetadata[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'playlistItemData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)